seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json

```diff
@@ -390,6 +390,117 @@
       "type": "backend",
       "challengeType": 0,
       "translations": {}
-    }  
+    },  
+    {
+      "id": "589a69f5f9fc0f352b528e6f",
+      "title": "Social Authentication with OAuth",
+      "description": [
+        [
+          "",
+          "",
```

Comment:
I think `challengeType` 7 is the new type used for the introduction- type of challenges. Their descriptions consist of arrays of 4 string, each having a different meaning like images for example.